albumentations/core/composition.py

```diff
@@ -253,6 +254,8 @@ def _check_args(self, **kwargs):
                 if data:
                     if not isinstance(data[0], np.ndarray):
                         raise TypeError("{} must be list of numpy arrays".format(data_name))
+            if internal_data_name in check_bbox_param and self.processors.get("bboxes") is None:
+                raise Exception("bbox_params must be specified for bbox transformations")
```

Comment:
Better to change `Exception` to `ValueError`.

Reply:
`Exception` changed to `ValueError`